### PR TITLE
[CBRD-24009] modified test case for Plan cache is not updated when view changes including inline view.

### PR DESCRIPTION
**Pull request by sy052**
refer to [http://jira.cubrid.org/browse/CBRD-24009](http://jira.cubrid.org/browse/CBRD-24009)

### CODE REVIEW

File: sql/_13_issues/_21_2h/cases/cbrd_24009.sql [hunk 1]
```diff
@@ -1,15 +1,21 @@
 drop table if exists tmp;
 create table tmp (col1 int);
 insert into tmp value(1),(1),(2);
+drop view if exists tmp_view;
 create or replace view tmp_view as select * from (select 1 from tmp where col1 = 2);
-select  count(*) from tmp_view;
+select count(*) from tmp_view;
+
+drop view if exists tmp_view;
```
Comment (shparkcubrid) [inline sql/_13_issues/_21_2h/cases/cbrd_24009.sql]: remove it.

File: sql/_13_issues/_21_2h/cases/cbrd_24009.sql [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
+drop view if exists vi;
 create table tmp (col1 int) DONT_REUSE_OID;
 create or replace view v1 as select * from tmp;
 
 INSERT INTO v1 SELECT * FROM v1 LIMIT 0;
 
 drop table if exists tmp;
-
+drop view if exists tmp_view;
+drop view if exists vi;
```
Comment (sjkimxxx) [inline sql/_13_issues/_21_2h/cases/cbrd_24009.sql]: vi -> v1

File: sql/_13_issues/_21_2h/cases/cbrd_24009.sql [hunk 3]
```diff
@@ -1,15 +1,20 @@
...
+drop view if exists tmp_view;
 create or replace view tmp_view as select * from (select 1 from tmp where col1 = 2);
-select  count(*) from tmp_view;
+select count(*) from tmp_view;
+
 create or replace view tmp_view as select * from (select 1 from tmp where col1 in ('1','2'));
 select count(*) from tmp_view;
+
 drop table if exists tmp;
+drop view if exists vi;
```
Comment (sjkimxxx) [inline sql/_13_issues/_21_2h/cases/cbrd_24009.sql]: The view name is wrong and it's not necessary.

File: sql/_13_issues/_21_2h/answers/cbrd_24009.answer [hunk 4]
```diff
@@ -26,3 +28,9 @@ count(*)
 0
 ===================================================
 0
+===================================================
+0
+===================================================
+0
+===================================================
+0
```
Comment (shparkcubrid) [inline sql/_13_issues/_21_2h/answers/cbrd_24009.answer]: no new line at the end of file.

**Review by ssihil — APPROVED** [review]

**Review by shparkcubrid — APPROVED** [review]

**Review by sjkimxxx — APPROVED** [review]

**Pull request merged**